src/software/ai/hl/stp/tactic/crease_defender/simulated_crease_defender_tactic_test.cpp

```diff
@@ -64,7 +64,7 @@ TEST_F(SimulatedCreaseDefenderTacticTest, test_chip_ball)
             Duration::fromSeconds(10));
 }
 
-TEST_F(SimulatedCreaseDefenderTacticTest, test_not_bumping_ball_towards_net)
+TEST_F(SimulatedCreaseDefenderTacticTest, DISTABLED_test_not_bumping_ball_towards_net)
```

Comment:
Why is this disabled? Please add a comment, also DISTABLED->DISABLED

Reply:
Re-enabled the test